test/specs/ellipse.spec.js

```diff
@@ -1,3 +1,16 @@
 describe('Ellipse annotation', function() {
   describe('auto', jasmine.fixtures('ellipse'));
+
+  window.testEvents({
+    type: 'ellispe',
```

Comment:
Took a closer look anyway and here is the problem with ellipse (its not ellispe) :)

Reply:
This leads me to think that we should at least output a warning when the type is not found.

Reply:
azz!!!! THANK YOU!

Reply:
fixed

Reply:
> This leads me to think that we should at least output a warning when the type is not found.

let me add it

Reply:
Lets do that separately